jar.go

```diff
@@ -396,6 +396,34 @@ func (j *Jar) deleteExpired(now time.Time) {
 	}
 }
 
+// RemoveAll removes any cookies from the jar that were set for the given host.
+func (j *Jar) RemoveAll(host string) {
+	now := time.Now().Add(-1 * time.Second)
```

Comment:
Good point.  I was going to do that and then never got around to it (another remnant of copy and paste).